addon/flash/object.js

```diff
@@ -1,13 +1,13 @@
 import Ember from 'ember';
 import customComputed from '../utils/computed';
-import computed from 'ember-new-computed';
 
 const {
   Object: EmberObject,
   run: { later, cancel },
   Evented,
   get,
-  set
+  set,
+  computed
```

Comment:
Ah, it looks we still need to pull in `computed` as well as `computed: { readOnly }`